src/Search_Replace_Command.php

```diff
@@ -479,7 +486,7 @@ private function php_handle_col( $col, $primary_keys, $table, $old, $new ) {
 		global $wpdb;
 
 		$count = 0;
-		$replacer = new \WP_CLI\SearchReplacer( $old, $new, $this->recurse_objects, $this->regex, $this->regex_flags, $this->regex_delimiter, null !== $this->log_handle );
+		$replacer = new \WP_CLI\SearchReplacer( $old, $new, $this->recurse_objects, $this->regex, $this->regex_flags, $this->regex_delimiter, $this->regex_limit, null !== $this->log_handle );
```

Comment:
Technically changing the order of the args to the `SearchReplacer` constructor is a BC break, but I think we can safely ignore this as it's unlikely to be used externally. Also I think this whole thing needs refactoring (passing so many args is a sign it's wrong) but that's a separate PR.